### PR TITLE
fix init-admin

### LINKED ISSUE #117: Missing instructions for setting up Admin
**Actual**
instructions:
```
cd webui
npm run init-admin <email>
```

user.ts:  
can't find 'credentials'
```
require('../credentials')
```
This logic isn't correct
```
if (userRecord.customClaims && userRecord.customClaims.role !== role)
```

*Updates*
instructions:
```
cd webui
npm i # or maybe just cypress
npm run init-admin <email>
```

user.ts:
```
if (!userRecord.customClaims || userRecord.customClaims!.role !== role)
```

### PR DESCRIPTION
**Pull request by zsxking**
fix errors when using the init-admin script.

fix #117

### CODE REVIEW

File: webui/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "prepare-e2e": "cypress/scripts/pass-gen.sh && ts-node cypress/scripts/prepareData.ts",
     "cleanup": "ts-node cypress/scripts/cleanup.ts",
     "report-coverage": "codecov",
-    "init-admin": "ts-node cypress/scripts/init-admin.ts"
+    "init-admin": "cypress/scripts/pass-gen.sh && ts-node cypress/scripts/init-admin.ts"
```
Comment (josueetcom) [inline webui/package.json:20]: `pass-gen.sh` creates `credentials.js`, should it be creating `credentials.ts`?

Comment (zsxking) [reply webui/package.json:20]: either is fine. The tsconfig is set to `allowJS`. I took the js one as it's easier when we ask users to create it manually. Now it's with the script, it's the same either way I think. But never bother to change it as it works just fine.

Comment (josueetcom) [reply webui/package.json:20]: Cool thanks.

**Pull request merged**